www/yarn.lock

```diff
@@ -12154,9 +12154,9 @@ socket.io-client@3.1.1:
     socket.io-parser "~4.0.4"
 
 socket.io-parser@~4.0.3, socket.io-parser@~4.0.4:
```

Comment:
Suggest consolidation of the ranges since the ranges overlap.
Suggested change `socket.io-parser@~4.0.4`